test/e2e/specs/martech/tos-screenshots__checkout.ts

```diff
@@ -120,16 +83,22 @@ describe( DataHelper.createSuiteTitle( 'ToS acceptance tracking screenshots' ),
 					type: 'jpeg',
 					quality: 20,
 				} );
-			}
-		} );
+			} );
+		}
+	);
+
+	it( 'Zip screenshots and upload', async function () {
+		const filetnameTitle = 'tos-screenshots-checkout';
+		const zipFilename = `${ filetnameTitle }.zip`;
+		const result = await uploadScreenshotsToBlog( zipFilename, 'tos_checkout_*' );
 
-		it( 'Zip screenshots and upload', async function () {
-			const filetnameTitle = 'tos-screenshots-checkout';
-			const zipFilename = `${ filetnameTitle }.zip`;
-			const result = await uploadScreenshotsToBlog( zipFilename, 'tos_checkout_*' );
+		expect( result?.media?.[ 0 ]?.title ).toStrictEqual( filetnameTitle );
+		expect( result?.media?.[ 0 ]?.mime_type ).toStrictEqual( 'application/zip' );
+	} );
 
-			expect( result?.media?.[ 0 ]?.title ).toStrictEqual( filetnameTitle );
-			expect( result?.media?.[ 0 ]?.mime_type ).toStrictEqual( 'application/zip' );
-		} );
+	afterAll( async function () {
+		await restAPIClient.clearShoppingCart(
```

Comment:
**Question: (non-blocking)**: do we want to check the response body here to make sure `success` is actually true? Would we want to alert some error if the actual clearing didn't go through beyond a request error?

Reply:
Sure, we can do that. Will update in next revision.